Source/WebCore/platform/graphics/filters/FEColorMatrix.cpp

```diff
@@ -134,6 +138,13 @@ std::unique_ptr<FilterEffectApplier> FEColorMatrix::createSoftwareApplier() cons
     return FilterEffectApplier::create<FEColorMatrixSoftwareApplier>(*this);
 }
 
+std::optional<GraphicsStyle> FEColorMatrix::createGraphicsStyle(const Filter&) const
```

Comment:
I don't think we'll ever get here for invert, opacity, brightness or contrast unless you also take bits of https://bugs.webkit.org/show_bug.cgi?id=245111

Also, I'd prefer that we write this in way where we can call functions on `ColorMatrix<>`. We should take parts of https://bugs.webkit.org/show_bug.cgi?id=245111 first.

Reply:
I agree but can we do this is a separate patch after landing https://bugs.webkit.org/show_bug.cgi?id=245111?

Reply:
OK